composer/trainer/dist_strategy.py

```diff
@@ -123,6 +123,49 @@ def prepare_ddp_module(module: torch.nn.Module, find_unused_parameters: bool) ->
                        'with distributed support.')
 
 
+def _recreate_fsdp_param_groups_from_unwrapped_opt_info(fsdp_wrapped_named_params: Iterator[Tuple[str,
```

Comment:
some of these names may be a little long so wanted to check with reviewers whether this is acceptable formatting